corehq/apps/case_importer/const.py

```diff
@@ -1,9 +1,10 @@
 from __future__ import absolute_import
 from django.utils.translation import ugettext_noop
+from six.moves import range
 
 
 class LookupErrors:
-    NotFound, MultipleResults = range(2)
+    NotFound, MultipleResults = list(range(2))
```

Comment:
We could get nitpicky about this specific instance, but I think nitpicking these would be a terrible idea.